surveys/2025/contributor-survey/questions.md

```diff
@@ -0,0 +1,181 @@
+# Survey questions
+
+This survey is organized by the Rust Leadership Council. Its goal is to figure out how much Rust Project contributions/work done by Rust Project members is funded vs. how much is volunteer-based. We want to use the results of this survey to secure more funding sources for Rust Project contributions in the future.
+
+This survey is only aimed at Rust Project members. If you are not a member of any Rust team/working group/project group/etc., please do not fill this survey. It is also primarily aimed at "active" contributors, who made at least some contribution in the past 12 months.
+
+Note that when we ask about contributions in this survey, we always mean contributions to the Rust Project repositories, so essentially any repository under the `rust-lang` GitHub organization, plus any other related organizations, such as `rust-lang-nursery`.
+
+### When have you started contributing to the Rust Project?
+
+Type: select one (optional)
+
+- During 2025
+- During 2023 or 2024
+- During 2021 or 2022
+- During 2020 or 2019
+- During 2018 or before
+- I do not contribute to the Rust Project [`NEXT`](<END>)
+
+### Are you funded for your Rust Project contributions?
+
+Type: select one
+
+- Yes, I am currently funded for (at least some of) my contributions [`NEXT`](#how-much-time-do-you-spend-contributing-to-the-rust-project-monthly)
+- No, but I was funded before [`NEXT`](#why-are-you-not-funded-anymore)
+- No, I never received any funding for them [`NEXT`](#would-you-appreciate-being-paid-for-contributing-to-the-rust-project)
+
+### Why are you not funded anymore?
+
+Type: select all that apply (optional)
+
+- I left an employment that funded my contributions
+- I have been laid off from an employment that funded my contributions
+- A contract that funded my contributions has not been renewed
+- A grant ran out
+- Individual sponsors (e.g. GitHub sponsors) stopped sponsoring me
+- I cannot receive funds anymore due to legislation (e.g. visa or student status) 
+- Other (open response)
+
+### How much time do you spend contributing to the Rust Project monthly?
+
+Estimate how many hours do you spend on Rust Project contributions each month. If this varies wildly, try to provide an average monthly duration over the past 12 months.
+
+This includes all kinds of contributions, such as programming, writing tests or documentation, triaging issues, investigating bugs, discussing on Zulip or GitHub, doing code reviews, etc.
+
+Enter the value as an integer, e.g. *50*.
+
+Type: free form (optional)
+
+### How much of your time spent on Rust Project contributions is funded?
+
+Estimate how much of the time that you spend on contributing to the Rust Project is funded. For example, if you spend 20 hours per week contributing, and a company pays you for 10 hours of contributions per week, then approx. 50% of your contribution time is funded.
```

Comment:
This is probably one of the more complex questions and might require a bit more guidance or examples of how to answer - I'm trying to think about how I'd answer it: I'm salaried for 38hr/wk, and all I work on are Rust contributions, but also have internal meetings and so on. Do I say that I spend 38hr/wk is contributing, do I remove ~5-6hrs/wk for the internal bits and say I'm contributing that much? It might be worth having some examples of common scenarios and how we'd like people to answer to encourage uniformity.

Reply:
Did you mean to put this to the "How much time do you spend contributing to the Rust Project monthly?" question?

For this question, we only want to know the ratio. So if you work 33hr/wk on Rust, then 100% of your time is funded, and you should answer 100.

Reply:
Oops, yeah, I intended to put this on "How much time do you spend contributing to the Rust Project monthly?"

Reply:
I meant examples in the sense of "common contributor funding arrangements" (salaried/contractor/etc) rather than "types of contribution" - e.g. if you're employed at a company and spend a few hours each week on internal meetings or 1-1s or whatever, you do/don't need to try and work out the actual number of hours contributing. Stuff like that, or similar things for other arrangements - just so that there's a little bit of standardisation as far as how people decide to tell us those details.

Reply:
Tried to clarify the meetings, let me know if you have suggestions for further edits!